### PR TITLE
made the final changes in Sanford Mysamples page

### PR DESCRIPTION
**Pull request by ramavavi**
The issue address is : https://github.com/episphere/connect/issues/688#issue-1835562887

The changes made in the code are:

- Removed the hours
- Added new information under What Happens subheading. 

Latest document for reference: 
[Sanford_My Samples page_08032023_tracked-2.docx](https://github.com/episphere/connectApp/files/12401141/Sanford_My.Samples.page_08032023_tracked-2.docx)

### CODE REVIEW

File: js/pages/samples.js [hunk 1]
```diff
@@ -313,26 +314,25 @@ const sanford = {
...
-    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples. Be sure you are scheduling an appointment in the city you are closest to.',
+    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.',
```
Comment (JoeArmani) [inline js/pages/samples.js:317]:
```suggestion
    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule an appointment at a time and location that is convenient for you.',
```
From this doc: https://nih.app.box.com/file/805344518276?s=e5xonzmm27rvd77crt1mnrbalit51a7v is this one current?

File: js/pages/samples.js [hunk 2]
```diff
@@ -313,26 +314,25 @@ const sanford = {
...
+    whatHappens: 'Check in at the registration desk. The registration team will direct you where to go next in order to get your samples collected.  At the end of your visit, the Connect team will check you out of your appointment.<br><br>We will draw a blood sample, collect a urine sample, and collect a saliva sample by asking you to swish with mouthwash.<br><br>We will also ask you to complete a short survey on the MyConnect app using your mobile phone. You will need your MyConnect app login information to complete the survey. If you do not have a mobile phone, we may be able to provide you with a tablet to complete your survey.<br><br>We strongly encourage you to complete your survey at your appointment. If you choose to complete it after you leave your appointment, it is important to do so as soon as possible.',
     support: 'Call 1-877-505-0253 (8:00 a.m.-10:00 p.m. CT on weekdays and 9:00 a.m.-6:00 p.m. CT on weekends)',
     locations: [
         [
-            'Sioux Falls: Sanford Imagenetics',
+            'Sanford Imagenetics',
```
Comment (JoeArmani) [inline js/pages/samples.js:325]:
```suggestion
            'Sioux Falls: Sanford Imagenetics',
```
It looks like the doc says: `Sioux Falls: Sanford Imagenetics` Please correct me if I'm wrong. https://nih.app.box.com/file/805344518276?s=e5xonzmm27rvd77crt1mnrbalit51a7v

File: js/pages/samples.js [hunk 3]
```diff
@@ -342,40 +342,34 @@ const marshfield = {
...
     whatHappens: 'The Connect team will check you in for your appointment and then collect your samples. At the end of your visit, the Connect team will check you out of your appointment.<br><br>We will draw a blood sample, collect a urine sample, and collect a saliva sample by asking you to swish with mouthwash.<br><br>We will also ask you to complete <span style="font-weight:900; text-decoration:underline">a short survey</span> on the MyConnect app using your mobile phone. If you do not have a mobile phone, we will provide a tablet for you to use complete your survey. You will need your MyConnect app login information to complete the survey.',
     support: 'Call 1-877-505-0253 (8:00 a.m.-10:00 p.m. CT on weekdays and 9:00 a.m.-6:00 p.m. CT on weekends)',
     locations: [
         [
             'Marshfield Clinic, Marshfield Center',
             '1000 N. Oak Ave<br>Marshfield, WI 54449',
-            'Monday - Thursday: 7:00 a.m. to 4:00 p.m. and Friday 7:00 a.m. to noon',
+            'Monday - Thursday: 7:00 a.m. - 4:00 p.m. and Friday 7a.m. - noon',
```
Comment (JoeArmani) [inline js/pages/samples.js:354]:
```suggestion
            'Monday - Thursday: 7:00 a.m. - 4:00 p.m. and Friday 7 a.m. - noon',
```
Just needs a space at `7 a.m.`

**Review by JoeArmani — CHANGES_REQUESTED** [review]
Just a few small changes or let me know if the doc version I was looking at is outdated. Thanks.

File: js/pages/samples.js (hunk 1 above)
Comment (JoeArmani) [reply js/pages/samples.js:317]: I reviewed the new doc - please update this per the suggestion.

File: js/pages/samples.js (hunk 2 above)
Comment (JoeArmani) [reply js/pages/samples.js:325]: I reviewed the new doc - please update this per the suggestion.

File: js/pages/samples.js [hunk 4]
```diff
@@ -313,11 +314,11 @@ const sanford = {
...
-    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples. Be sure you are scheduling an appointment in the city you are closest to.',
+    howToDonate: 'The MyChart message or email we send you will contain a link to schedule an appointment. Simply click the link to schedule an appointment at a time and location that is convenient for you.',
     scheduling: 'Scheduling link will be sent by the Connect team at Sanford.<br><br>For questions, please call 605-312-6100 or email <a href="mailto: connectstudy@sanfordhealth.org">connectstudy@sanfordhealth.org</a>.',
```
Comment (JoeArmani) [inline js/pages/samples.js:318]: From the doc: 
'Scheduling information will be sent by the Connect team at Sanford via your MyChart or email'

**Review by JoeArmani — APPROVED** [review]
looks good

File: js/pages/samples.js [hunk 5]
```diff
@@ -342,40 +342,34 @@ const marshfield = {
...
     whatHappens: 'The Connect team will check you in for your appointment and then collect your samples. At the end of your visit, the Connect team will check you out of your appointment.<br><br>We will draw a blood sample, collect a urine sample, and collect a saliva sample by asking you to swish with mouthwash.<br><br>We will also ask you to complete <span style="font-weight:900; text-decoration:underline">a short survey</span> on the MyConnect app using your mobile phone. If you do not have a mobile phone, we will provide a tablet for you to use complete your survey. You will need your MyConnect app login information to complete the survey.',
     support: 'Call 1-877-505-0253 (8:00 a.m.-10:00 p.m. CT on weekdays and 9:00 a.m.-6:00 p.m. CT on weekends)',
     locations: [
         [
             'Marshfield Clinic, Marshfield Center',
             '1000 N. Oak Ave<br>Marshfield, WI 54449',
-            'Monday - Thursday: 7:00 a.m. to 4:00 p.m. and Friday 7:00 a.m. to noon',
+            'Monday - Thursday: 7:00 a.m. - 4:00 p.m. and Friday 7 a.m. - noon',
```
Comment (anthonypetersen) [inline js/pages/samples.js:354]: removing the :00 breaks from the time style used everywhere else for this site, please confirm this is what's wanted as the document doesn't show

**Review by anthonypetersen — CHANGES_REQUESTED** [review]
feels like this PR is touching more than just the Sanford changes mentioned?

File: js/pages/samples.js (hunk 5 above)
Comment (ramavavi) [reply js/pages/samples.js:354]: it has to be 7:00 am, not 7 a.m. will make the change right away.

**Comment by ramavavi** [comment]
> feels like this PR is touching more than just the Sanford changes mentioned?

Looking into it.

**Pull request closed without merging**